mock/environment/__init__.py

```diff
@@ -27,19 +27,19 @@ def _data_file_path(filename: str) -> str:
         return os.path.join('..', '..', 'data', filename)
```

Comment:
This return is alone?

Reply:
Yeah, it's just a one-line method because we have to create four paths to files in the `data` path, so this just does it consistently.